### ISSUE TITLE
@looker/extension-sdk requires react

### ISSUE THREAD

**Issue by AndrewKoliaka** [looker-open-source/sdk-codegen#1438]
We use Looker in Angular project. And when I try to upgrade @looker/extension-sdk from 23.6.0 to the latest version 24.2.0 it breaks with the error:

```
ERROR in node_modules/@looker/extension-sdk/lib/connect/tile/types.d.ts:1:33 - error TS2307: Cannot find module 'react' or its corresponding type declarations.

1 import type { MouseEvent } from 'react';
```

**Comment by drstrangelooker** [comment]
@bryans99 Is this resolved by https://github.com/looker-open-source/sdk-codegen/pull/1532?

**Comment by jkaster** [comment]
This should be resolved by #1532
